### ISSUE TITLE
Prebid 1.0 targeting issues

### ISSUE THREAD

**Issue by bretg** [prebid/Prebid.js#1677]
## Type of issue
Targeting bugs in Prebid 1.0

## Description
1) When there's an invalid bidder, targeting isn't set for the valid bidders
2) enableSendAllBids isn't defaulting to true

## Steps to reproduce
1) build a prebid.js package for 1.0
2) set up a page with an AdUnit having a valid bidder and an invalid bidder
3) don't specify enableSendAllBids in setConfig

## Test page
```
    var adUnits = [
           {
               code: 'test-div',
               sizes: [[300,250],[300,600],[728,90]],
               bids: [
                   {
                       bidder: 'rubicon',
                       params: {
                           accountId: '1001',
                           siteId: '113932',
                           zoneId: '535510'
                       }
                   },{
                        bidder: 'appnexus',
                        params: {
                                placementId:'4799418'
                        }
                   }
               ]
           }
       ];
```
Note that 'appnexus' wasn't ported to 1.0 -- we have to use 'appnexusAst' now.

### Expected results

Targeting for the valid bidder is sent to the ad server. e.g.
```
hb_size_rubicon=728x90&hb_pb_rubicon=1.40&hb_adid_rubicon=3b79745d66736&hb_bidder_rubicon=rubicon&rpfl_1001=2_tier0100&hb_size=728x90&hb_pb=1.40&hb_adid=3b79745d66736&hb_bidder=rubicon
```
Note that there are two sets of targeting expected: one with the _rubicon suffix and one without.

### Actual results

1) When the invalid bidder is present, no targeting is sent at all.

2) When the invalid bidder is removed (or fixed to appnexusAst), the targeting goes out, but by default is sending only the non-suffixed keys:
```
hb_size=728x90&hb_pb=1.40&hb_adid=5f52cc59e2faf5&hb_bidder=rubicon
```

## Platform details
Prebid 1.0

## Other information
Package was built with 
```
gulp build-bundle-dev --modules=rubiconBidAdapter,appnexusAstBidAdapter
```

**Comment by matthewlane** [comment]
Was this possibly fixed with #1970?

**Comment by matthewlane** [comment]
Please reopen if #1970 didn't fix this
